README.md

```diff
@@ -129,9 +129,18 @@ The easiest way to spin up all components described above is to use:
 [Podman]: https://podman.io/
 [Podman Compose]: https://github.com/containers/podman-compose
 
+To install latest stable version on Ubuntu you can use `brewlinux`:
+```shell
+sudo apt-get remove crun podman
+brew install podman
+brew install crun podman
+pip3 install podman-compose
+```
+
 ### Localnet deployment
 
 ```shell
+mkdir -p ./contracts/out ./contracts/cache && chmod -R 777 ./contracts
```

Comment:
Running `chmod -R 777` is a very bad idea.

I investigated further the permissions issue and I think adding `.dockerignore` whould be a better approach. See https://github.com/oasisprotocol/demo-rofl-chatbot/issues/12#issuecomment-2858318765

Reply:
Removed permission instructions, added .dockerignore.